auslib/test/admin/views/test_releases.py

```diff
@@ -684,7 +684,7 @@ def testPostRevisionRollback(self):
         assert row['name'] == 'd'  # one of the fixtures
 
         url = '/releases/d/revisions'
-        ret = self._post(url, {'change_id': change_id})
+        ret = self._post(url, json.dumps({'change_id': change_id}), content_type="application/json")
```

Comment:
Maybe a JSONTestMixin._post() would be helpful to avoid repetition of content_type.